src/Apps/MyCollection/Routes/EditArtwork/Utils/useCreateOrUpdateArtwork.ts

```diff
@@ -65,7 +66,9 @@ const formValuesToMutationInput = (
     values.editionSize = ""
   }
 
-  const externalImageUrls = values.newPhotos.flatMap(photo => photo.url || null)
+  const externalImageUrls = compact(
+    values.newPhotos.flatMap(photo => photo.url || null)
+  )
```

Comment:
Adding `compact` here to avoid `null` values that would cause the mutation to fail.